libs/sdk-ui-pivot/styles/scss/pivotTable.scss

```diff
@@ -273,7 +273,6 @@ $table-menu-hover: var(
 
         .gd-cell-show-hidden {
             color: inherit;
-            font-weight: normal;
```

Comment:
Why was this removed?

Reply:
It caused an issue with missing Bold font style (when drilling on elements is active), when last item in group of the same elements is rendered for smoother scroll out from the view 
https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-pivot/src/impl/stickyRowHandler.ts#L91
Found accidentally

Reply:
Oh ok, thanks :)